casts/bars/src/UILargeTitleHeader/useRubberBandEffectDistance.tsx

```diff
@@ -0,0 +1,26 @@
+import { Platform } from 'react-native';
+import { useDerivedValue } from 'react-native-reanimated';
+
+import { runOnUIPlatformSelect } from './useScrollHandler/runOnUIPlatformSelect';
+
+// @inline
+const RUBBER_BAND_EFFECT_DISTANCE_WEB = 50;
+// @inline
+const RUBBER_BAND_EFFECT_DISTANCE_DEFAULT = 100;
+
+export const NON_UI_RUBBER_BAND_EFFECT_DISTANCE = Platform.select({
+    web: RUBBER_BAND_EFFECT_DISTANCE_WEB,
+    default: RUBBER_BAND_EFFECT_DISTANCE_DEFAULT,
+});
```

Comment:
I wonder if it looks well on mobile web (probably it's not so important, but can be tested just in case)

Reply:
I actually never did large title header properly for mobile web (didn't support touch events instead of wheel), so it might not work by design :D

Reply:
OK :)